src/fighter_state.rs

```diff
@@ -274,6 +285,9 @@ impl Dying {
     pub const ANIMATION: &'static str = "dying";
 }
 
+#[derive(Component)]
+pub struct BeingHold;
```

Comment:
I think maybe this should be:

```suggestion
pub struct BeingHeld;
```